src/qlipperpreferences.cpp

```diff
@@ -19,6 +19,9 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #include "qlipperpreferences.h"
 
+QString DEFAULT_ICON_PATH = ":/icons/qlipper.png";
```

Comment:
- why isn't it `const`?
- how about using `QStringLiteral`?
- when it's needed elsewhere, why not put it into as a static member of `QlipperPreferences`?